dietpi/dietpi-software

```diff
@@ -11174,6 +11213,54 @@ _EOF_
 
 		fi
 
+		software_id=177 # Firefox Sync Server
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+
+			# User
+			local usercmd='useradd -rMU'
+			getent passwd ffsync &> /dev/null && usercmd='usermod -a'
+			$usercmd -G dietpi -d /opt/ffsync -s $(command -v nologin) ffsync
+
+			# Create database directory
+			mkdir -p $G_FP_DIETPI_USERDATA/firefox-sync-db
+
+			# Create database if it doesn't exist
+			if [[ ! -a $G_FP_DIETPI_USERDATA/firefox-sync-db/FF-Sync-DB.db ]]; then
+				sqlite3 $G_FP_DIETPI_USERDATA/firefox-sync-db/FF-Sync-DB.db "create table aTable(field1 int); drop table aTable;"
+			fi
+
+			# Modify config file
+			ffsync_ip=$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1 -d'/')
+			dietpiUsrDataEsc=$(sed 's|/|\\/|g' <<< $G_FP_DIETPI_USERDATA)
+			sed -i.bak "s/host = 0.0.0.0/host = $ffsync_ip/" /etc/firefox-sync/syncserver.ini
```

Comment:
Ah here we have the config file. This cannot be moved to a different location outside the actual program/install dir, can it?